datafusion/src/logical_plan/expr.rs

```diff
@@ -934,6 +934,19 @@ impl Not for Expr {
     }
 }
 
+impl std::fmt::Display for Expr {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self {
+            Expr::BinaryExpr {
+                ref left,
+                ref right,
+                ref op,
+            } => write!(f, "{} {} {}", left, op, right),
+            _ => write!(f, "{}", ""),
```

Comment:
Ideally, this code would cover all the variants of `Expr` -- however, that doesn't have to be done in one PR. 

How about as a potential interim solution we could do something like the following and use the debug display until someone has time to add a better display implementation?

```suggestion
            _ => write!(f, "{:?}", self),
```